base_generate_code/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module can be plugged to other models, in order to generate random codes (e.g. for customer's purposes : coupon code, Gift card code, etc.).
```

Comment:
The goal is to have human readable code: not too long, random enough, without some ambiguous char like ( "O" "0", "I", "L"), not memorable, without meaning (no words), acceptable as a barcode (like code128).